README.md

```diff
@@ -31,19 +31,37 @@ your account to the `libvirtd` group.
 
 
 # Capabilities
-* **boot2docker.iso** based images
-* **Dual Network**
-    * **eth1** - A host private network called **docker-machines** is automatically created to ensure we always have connectivity to the VMs.  The `docker-machine ip` command will always return this IP address which is only accessible from your local system.
-    * **eth0** - You can specify any libvirt named network.  If you don't specify one, the "default" named network will be used.
+
+## Images
+By default `docker-machine-kvm` uses a [boot2docker.iso](https://github.com/boot2docker/boot2docker) for the kvm host. It's also possible to use every image that is derived from [boot2docker.iso](https://github.com/boot2docker/boot2docker) as well.
```

Comment:
Thanks 👍  for this you are right it I didn't use the correct terms in this context. I did correction in a separate commit https://github.com/dhiltgen/docker-machine-kvm/pull/25/commits/ae3b870f39b133e639388c60b5e3ddbbc9afdc9c . I will squash it if your review was okay .